README.md

```diff
@@ -255,10 +257,20 @@ Hence, you have to make sure new versions of `flex` and `bison` executables are
 system path as:
 ```
 export PATH="/usr/local/opt/flex/bin:/usr/local/opt/bison/bin:$PATH"
-```
+``` 
 Finally, in order to get the Lydia parser working, you may have to set the 
 following environment variable up (e.g. in your `.bashrc`) as follows:
 
 ```
 export CPLUS_INCLUDE_PATH="/System/Volumes/Data/usr/local/Cellar/flex/2.6.4_1/include/:$CPLUS_INCLUDE_PATH"
 ```
+
+## License
+
+This software is released under the GNU Lesser General Public License version 3 or later.
+
+## Acknowledgements
+
+This work has been funded by the [ERC Advanced Grant "WhiteMech"](whitemech.github.io/)
+and by the [TAILOR research network](https://tailor-network.eu/).
+
```

Comment:
@francescofuggitti please let me know your thoughts, and please feel free to change as you see fit!

Reply:
I would say "has been partially funded". Then, we could be precise with grants numbers, but I think it is also ok as is